srcmedia/scss/members/_map.scss

```diff
@@ -0,0 +1,32 @@
+section.map  {
+    @include container;
+    margin-bottom: 2rem;
+
+    hr {
+        width: 100%;
+        margin: 0;
+        border: none;
+        background-color: $white-two;
+        height: 1px;
+    }
+
+    h2 {
+        margin: 2rem 0 1rem;
+        padding-left: 1rem;
+        font-family: $sans;
+        font-size: 0.8rem;
+    }
+
+    #address-map {
+        width: 100%;
+        height: 415px;
```

Comment:
Why are the heights set absolutely in pixels?

Reply:
so, this reason for this is just that i forgot to convert them to rem - good catch. unfortunately while reading about applying rems to things like image/iframe sizing i came across [this piece](https://mindtheshift.wordpress.com/2015/04/02/r-i-p-rem-viva-css-reference-pixel/) which confirms something i was suspicious about...rems have basically no effect re:accessibility in modern browsers anymore :(

basically browsers now handle zooming based on a "reference pixel" that the compositor uses to paint the page, so if you scale that pixel, you can scale the entire page equally without needing to resort to responsive measurements. apparently almost all browsers have been using this approach for the zoom controls (<kbd>cmd</kbd><kbd>+</kbd>, etc) for awhile now, and have relegated "text-only zoom" to its own, more hidden (or nonexistent) option/menu.

this kind of throws a wrench in my conception of the utility of these responsive measurements. someone did bring up in the comments my initial thought, which was that even if the user doesn't use rems for scaling, the developer can use them to scale the whole typography system up/down via media queries just by setting the value on the root element, but people made the (probably valid) point that type design doesn't really work like that...on different sizes you have to _independently_ adjust headers vs body sizing to get the best fit (which is exactly what gissoo does), you can't just scale them together and hope they look good. so...maybe there isn't really much utility to rem/em after all.

anyway, this is all to say - i'll change these to rem, and we can continue using it in s&co since that's the foundation we started it with, but i guess we might want to have future projects just use straight up pixel values (which is in a way simpler since I can get them straight from zeplin).